app/components/application_component.rb

```diff
@@ -23,4 +23,13 @@ def dom_id(*args, **kwargs)
     return if args.blank?
     super
   end
+
+  def policy(*args, **kwargs)
+    Pundit.policy(current_person, *args, **kwargs)
+  end
+
+  def current_person
+    @current_person ||= helpers.current_person
+  end
+  attr_writer :current_person
```

Comment:
Yes, this PR is `bigger than it probably should be`. Please break it up into small PRs if possible for easier reviewing and 🧠 contextual absorption. I recommend the Kent Beck [method](https://medium.com/@kentbeck_7670/bs-changes-e574bc396aaa) of splitting it into behavior or structure changes.

> there are two kinds of changes — behavior changes and structure changes. Always be making one kind of change or the other, but never both at the same time.

Reply:
Will do!